src/App.tsx

```diff
@@ -5,7 +5,13 @@ import Footer from "./components/organisms/Footer/Footer";
 import Gallery from "./components/organisms/Gallery/Gallery";
 
 const App = () => {
-  const [data, setData] = useState<any>();
+  type JSONData = {
+    about: any
+    gallery: any
+    footer: any
```

Comment:
why any? you have data types already implemented in your components. import them and assign accordingly

Reply:
Changed